### PR TITLE
Node beta example

### PR DESCRIPTION
**Pull request by kevinpagtakhan**
### Summary

Adds example for Node.js express application

### Checklist

* [ ] Does your PR title have the correct [title format](https://github.com/amplitude/Amplitude-TypeScript/blob/main/CONTRIBUTING.md#pr-commit-title-conventions)?
* Does your PR have a breaking change?:

### CODE REVIEW

File: examples/node/express-app/bin/www [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+ *
+ * Calling init() requires an API key
+ * ```
+ * amplitude.init(API_KEY)
+ * ```
+ *
+ * Optionally, a config object can be provided. Refer to https://amplitude.github.io/Amplitude-TypeScript/interfaces/Types.BrowserConfig.html
+ * for object properties.
+ */
+amplitude.init('API_KEY');
```
Comment (liuyang1520) [inline examples/node/express-app/bin/www:41]: Just my idea, feel free to ignore or not. Maybe updating this to `amplitude.init(process.env.API_KEY || 'API_KEY');`, so after cloning the repo, no need to update the code, can probably start with `API_KEY=<api-key> npm start`.

**Review by liuyang1520 — COMMENTED** [review]
Do we want to add a README file for this new example?

**Comment by yuhao900914** [comment]
+1 for the readme. 
Just noticed we only have the init and track functions. Are we planning to add more examples like group or identify, or even the plugin?

File: examples/node/express-app/bin/www (hunk 1 above)
Comment (dnshi) [reply examples/node/express-app/bin/www:41]: Interesting point! I like the idea.

**Review by dnshi — APPROVED** [review]
It looks good to me 💯. Thanks for adding the Node examples.

**Review by yuhao900914 — COMMENTED** [review]
Is this still a valid pr?

**Pull request closed without merging**